### ISSUE TITLE
Panther not parsing homologues parameter correctly

### ISSUE THREAD

**Issue by julie-sullivan** [intermine/intermine#1079]
@kkarra is not seeing human orthologues but I see them just fine:

```
flymine=# select o.taxonid, count(*) from gene g, organism o where g.organismid = o.id group by o.taxonid;
 taxonid | count 
---------+-------
    9606 | 16561
    4932 |  4420
(2 rows)
```

**Comment by julie-sullivan** [comment]
@kkarra I just figured out what was different! And why mine worked and yours didn't.

This is what my project XML looks like:

```
   <source name="panther" type="panther" dump="true">
      <property name="src.data.dir" location="/micklem/data/panther/current"/>
      <property name="panther.organisms" value="4932 9606"/>
    </source>
```

You have a orthologues parameter. Change to use my config if you want a quick fix. I am working on this now.

**Comment by julie-sullivan** [comment]
Fixed by 3a6fb540c211b1e0626f99a07e4209464908ad6c

**Comment by julie-sullivan** [comment]
@justinccdev tested this, it is working as expected. @kkarra if you are still having problems let me know and I'll reopen the ticket.
